report/collect.go

```diff
@@ -96,7 +96,7 @@ func detectContentType(path string) string {
 	}
 	defer func() {
 		if err := file.Close(); err != nil {
-			// This is empty on purpose to please the linter
+			panic(err)
```

Comment:
This alter the behaviour significantly because it will fail the step. Previously the worst case was that we most probably just leaked some memory. 

What if we just simply do 
```
_ = file.Close()
```
and annotate it so the linter does not complain about it?

Reply:
Good point, thank you! The leak probably doesn't matter in these short-lived step processes anyway.